tests/pkcli/github_test.py

```diff
@@ -45,18 +45,20 @@ def test_issue_start():
     github.issue_pending_alpha(github._TEST_REPO)
     with pkunit.pkexcept('prior release'):
         github.issue_start_alpha(github._TEST_REPO)
-    i, t = _create_commit(r)
-    a = github.issue_update_alpha_pending(github._TEST_REPO)
-    pkunit.pkre(i, a)
-    pkunit.pkre(t, a)
+    for b in (False, True):
+        i, t = _create_commit(r, full_name=b)
+        f = f'{r.full_name}#{i}'
```

Comment:
You want to test both cases, and this only tests, one. We want to see that both issues got added.

This is when I would make it a nested function, something like (see changes below):

```py
def _commit(repo, full_name, issues):
    issues.append(_create_commit(repo, full_name=full_name))
    a = github.issue_update_alpha_pending(github._TEST_REPO)
    pkunit.pkre(issues[-1].title, a)
    pkunit.pkre(issues[-1].link, a)
    
issues = []
_commit(r, False, issues)
_commit(r, True, issues)
```

Then `issues` will be used below.